package.json

```diff
@@ -14,12 +14,14 @@
   },
   "dependencies": {
     "@babel/plugin-transform-react-jsx": "^7.10.4",
+    "chart.js": "2.9.3",
```

Comment:
Please remove the un-necessary dependencies

Reply:
So, I observed , react chart js was facing issue without chart js , both are dependent
Or required